plugins/modules/source_control/gitlab/gitlab_group_variable.py

```diff
@@ -0,0 +1,295 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Florent Madiot (scodeman@scode.io)
+# Based on code:
+# Copyright: (c) 2019, Markus Bergholz (markuman@gmail.com)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+module: gitlab_group_variable
+short_description: Creates/updates/deletes GitLab Groups Variables
+version_added: 1.2.0
+description:
+  - When a group variable does not exist, it will be created.
+  - When a group variable does exist, its value will be updated when the values are different.
+  - Variables which are untouched in the playbook, but are not untouched in the GitLab group,
+    they stay untouched (I(purge) is C(false)) or will be deleted (I(purge) is C(true)).
```

Comment:
I find this paragraph hard to understand. Can you try to rewrite it, or split it up into easier sentences?

Reply:
this is the same paragraph as for the current [gitlab_project_variable.py](https://github.com/ansible-collections/community.general/blob/main/plugins/modules/source_control/gitlab/gitlab_project_variable.py), I did not change any of it to be consistent as it applies the same way to groups.